src/element.ts

```diff
@@ -67,26 +88,29 @@ export class Element {
     return this;
   }
 
-  restoreStyle(context = this.context, style = this.getStyle()) {
+  restoreStyle(context: RenderContext | undefined = this.context, style: Style | undefined = this.getStyle()): this {
```

Comment:
```ts
context?: RenderContext
```

Reply:
This is not possible, I believe,  with a default parameter ( = this.context ).

Reply:
Maybe `context: RenderContext = this.context` is enough, no? It breaks if there is no context. (i.e., no need for `undefined`, let the compiler verify that.)

Reply:
@wassertim _"Parameter cannot have question mark and initializer.ts(1015)"_
@0xfe `this.context` can also be undefined, it is not initilized in the constructor: _"Type 'undefined' is not assignable to type 'RenderContext'.ts(2322)"_